remote_engine/src/mock_impl.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+//! Mock impl for remote table engine
+
+use async_trait::async_trait;
+pub struct MockImpl;
+
+#[async_trait]
+impl RemoteEngine for MockImpl {
```

Comment:
Maybe this trait should be defined in the `table_engine` crate?

Reply:
Really a good idea.